bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/codemining/LineHeaderCodeMining.java

```diff
@@ -76,23 +72,4 @@ public LineHeaderCodeMining(Position position, ICodeMiningProvider provider, Con
 			throws BadLocationException {
 		super(position, provider, action);
 	}
-
-	@Override
-	public Point draw(GC gc, StyledText textWidget, Color color, int x, int y) {
```

Comment:
I assume this method (with the exact same signature9 is now inherited from the new super-class?
Is this a compatible chang?